sdk/containerservice/arm-containerservice/review/arm-containerservice.api.md

```diff
@@ -455,9 +574,6 @@ export interface IstioComponents {
 // @public
 export interface IstioEgressGateway {
     enabled: boolean;
-    nodeSelector?: {
-        [propertyName: string]: string;
-    };
```

Comment:
what's the swagger change ?

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/27960